internal/db/schema/migrations/oss/postgres/55/03_create_kms_key_destructions.up.sql

```diff
@@ -0,0 +1,109 @@
+begin;
+
+  create table kms_data_key_version_destruction_job (
+    key_id kms_private_id primary key -- one job per key version
+      references kms_data_key_version (private_id)
+        on delete cascade -- Note: instead of restrict, we cascade the delete to this job when the key version is deleted
+        on update cascade,
+    create_time wt_timestamp not null
+  );
+  comment on table kms_data_key_version_destruction_job is
+    'Table holding running and pending data key version destruction jobs';
+
+  create trigger immutable_columns before update on kms_data_key_version_destruction_job
+    for each row execute procedure immutable_columns('key_id', 'create_time');
+
+
+  create table kms_data_key_version_destruction_job_run (
+    key_id kms_private_id not null
+      references kms_data_key_version_destruction_job (key_id)
+        on delete cascade
+        on update cascade,
+    table_name text not null,
+    total_count int not null
```

Comment:
Would it be helpful if a trigger checked (or populated) `total_count`?

Reply:
I think we should keep doing this in the application, as it lets us avoid adding rows that have no rows that need rewrapping.

Reply:
If we wanted to check that total count is accurate, we would need to use postgres' eval function to count rows in the table name (since it is a parameter).